devon_agent/agent.py

```diff
@@ -61,11 +62,13 @@ def _format_editor_entry(self, k, v, PAGE_SIZE=50):
             [str(i + start_idx).zfill(4) + line for i, line in enumerate(lines)]
         )
 
-        return f"""
-************ FILE: {path}, WINDOW STARTLINE: {start_idx}, WINDOW ENDLINE: {start_idx+content_len}, TOTAL FILE LINES: {all_lines_len} ************
-{window_lines}
-************************************
-"""
+        return textwrap.dedent(
```

Comment:
Prompts are extremely sensitive to even whitesoace tokens. Will have to make sure there is no regression

Reply:
I know, technically there shouldn't have been any changes, however I will wait for updated.